packages/ethereum/ethereum-light-client/src/error.rs

```diff
@@ -27,11 +27,13 @@ pub enum EthereumIBCError {
     #[error("insufficient number of sync committee participants ({0})")]
     InsufficientSyncCommitteeParticipants(u64),
 
-    #[error("update header contains deneb specific information")]
-    MustBeDeneb,
+    #[error("unsupported fork version, we only support electra")]
```

Comment:
I feel like this should be MustBeElectra, or we will forget to update this string

Reply:
Yeah, maybe that's better, will change to that!